### ISSUE TITLE
Update Rails tutorial to 0.6.0

### ISSUE THREAD

**Issue by solnic** [rom-rb/rom-rb.org#42]
Since 0.6.0 is around the corner we should update tutorial before pushing the release so the site can be updated quickly

- [x] update to use sub-classes of relation and mappers
- [x] update to use Form objects (skip commands completely)

**Comment by maetl** [comment]
Can someone link to a Gist or quick example of form objects?

**Comment by solnic** [comment]
@maetl see ie [here](https://github.com/rom-rb/rom-rails/blob/master/spec/dummy/app/models/update_user_form.rb) in rom-rails + other specs etc.

If something's unclear I can give more examples :)

**Comment by solnic** [comment]
...I guess the crucial bit is this:

``` ruby
class NewUserForm < ROM::Form
  commands users: :create # <=== this defines commands for this form automatically so you have
                                           #           `users[:create]` command available in the form object
  # stuff
end
```

**Comment by maetl** [comment]
Made a start on this in #48.

**Comment by solnic** [comment]
Awesome thank you! :)

**Comment by solnic** [comment]
btw when rom-rb/rom-rails#25 is merged generators will be producing sub-classes

**Comment by maetl** [comment]
Initial version of the new tutorial with forms was merged with a8d6ccdceeb5491297e6497824d1c77b09a027d5.
